go/chat/server.go

```diff
@@ -2266,6 +2266,29 @@ func (h *Server) SearchInbox(ctx context.Context, arg chat1.SearchInboxArg) (res
 		}
 	}()
 
+	// send up team name matches
+	teamUIDone := make(chan struct{})
```

Comment:
why dont' we move this to an `errgroup.Group` or similar primitive

Reply:
I don't think we should be refactoring existing code as part of this miami work.  It's a good suggestion but I don't think we should complicate this PR with that.  This PR is just for plumbing.

Reply:
Ah makes sense